packages/lime-system/files/usr/lib/lua/lime/wireless.lua

```diff
@@ -82,6 +85,20 @@ function wireless.mesh_ifaces()
 				table.insert(ifaces, entry.ifname)
 			end
 		end)
+	--add apup interfaces 
+	--this are not listed in uci 
+	local shell_output = utils.unsafe_shell("ls /sys/class/net/ -R")
```

Comment:
this line made the hole testing framework to explode, but it seems that the problem was in lime location tests. I improved the stub handling in this file and the test framework started working again.